### ISSUE TITLE
apt.syncthing.net lists unavailable version

### ISSUE THREAD

**Issue by martinlindhe** [syncthing/syncthing#4290]
```
$ sudo apt-get upgrade
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
The following packages have been kept back:
  linux-headers-4.11.0-2-amd64
The following packages will be upgraded:
  syncthing
1 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
Need to get 6,434 kB of archives.
After this operation, 1,997 kB of additional disk space will be used.
Do you want to continue? [Y/n]
Err:1 https://apt.syncthing.net syncthing/stable amd64 syncthing amd64 0.14.34
  404  Not Found [IP: 46.101.130.230 443]
E: Failed to fetch https://apt.syncthing.net/dists/syncthing/stable/binary-amd64/syncthing_0.14.34_amd64.deb  404  Not Found [IP: 46.101.130.230 443]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

**Comment by calmh** [comment]
Yes, this is a temporary situation due to #4288 that will be resolved when a 0.14.35 is released shortly.

**Comment by martinlindhe** [comment]
@calmh Okay, thanks for the information :-)
